client/app/bundles/course/assessment/submission/pages/QuestionIndex/PastAttempts.tsx

```diff
@@ -0,0 +1,50 @@
+import { FC } from 'react';
+import { useParams } from 'react-router-dom';
+import { QuestionType } from 'types/course/assessment/question';
+import { QuestionAllAnswerDisplayDetails } from 'types/course/statistics/assessmentStatistics';
+
+import { fetchAllAnswers } from 'course/assessment/operations/statistics';
+import AllAttemptsDisplay from 'course/assessment/pages/AssessmentStatistics/AnswerDisplay/AllAttemptsDisplay';
+import Comment from 'course/assessment/pages/AssessmentStatistics/AnswerDisplay/Comment';
+import LoadingIndicator from 'lib/components/core/LoadingIndicator';
+import Preload from 'lib/components/wrappers/Preload';
+import { getEditSubmissionQuestionURL } from 'lib/helpers/url-builders';
+import { getSubmissionQuestionId } from 'lib/helpers/url-helpers';
+
+const PastAnswers: FC = () => {
+  const submissionQuestionId = getSubmissionQuestionId();
```

Comment:
Can't you use the `useParams()` hook?  
`const { courseId, assessmentId, submissionQuestionId } = useParams();`

Don't need to create unnecessary url-helpers.

Reply:
done